test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs

```diff
@@ -95,10 +96,11 @@ async IAsyncEnumerable<Page<BinaryData>> CreateEnumerableAsync(string nextLink,
         /// <summary> A paging operation that must return result of the default &apos;value&apos; node. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
         /// Schema for <c>Response Body</c>:
         /// <code>{
-        ///   value: [Product],
-        ///   nextLink: string
+        ///   value: [Product], # Optional.
```

Comment:
This should return the schema for a single element in the collection that is returned.